medusa/post_processor.py

```diff
@@ -1040,7 +1041,7 @@ def process(self):
                    (common.Quality.qualityStrings[new_ep_quality]), logger.DEBUG)
 
         # see if this is a priority download (is it snatched, in history, PROPER, or BEST)
-        priority_download = self._is_priority(old_ep_quality, new_ep_quality)
+        priority_download = self._is_priority(old_ep_quality, new_ep_quality) or self.manually_searched
```

Comment:
This will cause the file to be processed over and over if it's not moved away from the pp folder.

Reply:
Ok so your suggesting that we need to keep track of an additional flag. Which is enabled when it has been processed once?

Reply:
Sorry, I was actually wrong, it should work just fine in `post_processor` now. The issue was rather in `process_tv`. Should be good now.